lib/instadeploy.py

```diff
@@ -184,16 +186,26 @@ def build():
     if response.status_code != requests.codes.ok:
         raise MxBuildFailure("MxBuild failure", response.status_code, response.json())
 
+    result = response.json()
+    if result['status'] == 'Success':
+        try:
+            sync_project_files()
+            logger.info("Syncing project files ...")
+        except:
+            logger.warning("Syncing project files failed: %s", traceback.format_exc())
+    else:
+        logger.warning("Not syncing project files. MxBuild result: %s", result)
+
+    return result
+
+def sync_project_files():
```

Comment:
That's why choosing readable function names are as important.

If we look from another side, we also have the cost and time for understanding the `for` loop
for somebody who wants to make a high-level understanding of the code. With using 
functions, I agree that the code becomes larger, but reading it would be like 
(a) do build (b) if success, do sync (c) otherwise raise error or issues

Without the function, I need to first read the code to see that `rsync` is used for
the purpose of syncing of project files after building.

This is my stand and perspective generally. I can change the code in the next PR
if you're still not convinced.